docs: Add open collective in readme (#2948)
excalidraw/excalidraw: README.md

      <img src="https://badges.crowdin.net/excalidraw/localized.svg">
    </a>
  </p>

>   <p?>Ask questions or hang out on our  <a target="_blank" href="https://discord.gg/UexuTaE">discord.gg/UexuTaE</a>.</p>

Reviewer: ```suggestion
  <p>Ask questions or hang out on our <a target="_blank" href="https://discord.gg/UexuTaE">discord.gg/UexuTaE</a>.</p>
```

Read the latest news and updates on our [blog](https://blog.excalidraw.com). A good start is to see all the updates of [One Year of Excalidraw](https://blog.excalidraw.com/one-year-of-excalidraw/).

## We accept donations

> If you like the project and your company is using it you can become a contributor from [open collective](https://opencollective.com/excalidraw).

Reviewer: ```suggestion
If you like the project, you can become a sponsor at [Open Collective].(https://opencollective.com/excalidraw).
```

let's not exclude individual sponsors, too :)

Read the latest news and updates on our [blog](https://blog.excalidraw.com). A good start is to see all the updates of [One Year of Excalidraw](https://blog.excalidraw.com/one-year-of-excalidraw/).

## We accept donations

> If you like the project, you can become a sponsor at [Open Collective](https://opencollective.com/excalidraw).

Reviewer: ```suggestion
If you like the project, you can contribute to our costs of running the infrastructure by becoming a sponsor at [Open Collective](https://opencollective.com/excalidraw).
```

maybe, so that new users who may not be aware that Excalidraw has a backend that costs money?

Author: mixed feeling for that.. dunno who doesn't know that things needs electricity to keep it running :)

Author: I liked the first shorter version better.

Reviewer: I meant, it's not immediately obvious we have WS and Storage servers. But let's leave it then.
